HybrasylXML/XSD/Castables.xsd

```diff
@@ -92,8 +93,8 @@
         <xs:list itemType="hyb:Class" />
       </xs:simpleType>
     </xs:attribute>
-    <xs:attribute name="Id" type="xs:int"/>
-    <xs:attribute name="Speed" type="xs:int"/>
```

Comment:
Why the change on types?

Reply:
To match extant datatypes (int/speed are not ints in Hybrasyl itself).